runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -176,7 +176,7 @@ public long reauthorize(
                 .orElse(null);
 
             JwtSessionStore sessionStore = supplySessionStore(contextId);
-            session = sessionStore.supplySession(subject, roles);
+            session = sessionStore.supplySession(subject, credentials, roles);
```

Comment:
I think we only need to key the session by subject and roles, but retain the credentials as a field below, like other session attributes.

This will also ensure that latest credentials are in effect after reauthorizing an existing session, as older credentials may have expired.